test/integration/targets/collections_runtime_pythonpath/ansible-collection-python-dist-boo/README.rst

```diff
@@ -0,0 +1,107 @@
+.. image:: https://img.shields.io/pypi/v/ansible-collections.python.dist.boo.svg?logo=Python&logoColor=white
```

Comment:
Please omit unnecessary files from the test. The additional files distract from the actual test and make it much more difficult to see what is actually being tested.